src/applications/personalization/profile-2/components/personal-information/AddressesTable.jsx

```diff
@@ -1,71 +1,26 @@
 import React from 'react';
-import { connect } from 'react-redux';
 
-import { selectVet360Field } from 'platform/user/profile/vet360/selectors';
-
-import AddressView from 'platform/user/profile/vet360/components/AddressField/AddressView';
-import { FIELD_NAMES } from 'platform/user/profile/vet360/constants';
-import prefixUtilityClasses from 'platform/utilities/prefix-utility-classes';
+import MailingAddress from 'platform/user/profile/vet360/components/VAPMailingAddress';
+import ResidentialAddress from 'platform/user/profile/vet360/components/VAPResidentialAddress';
 
 import ProfileInfoTable from '../ProfileInfoTable';
 
-const ContactInfoCell = props => {
```

Comment:
I was able to get rid of this fake read-only data I had put in place and replace each address field with a single `VAP Address` component